src/components/mobileMenu/ExpandComponent.js

```diff
@@ -2,6 +2,7 @@ import expandArrow from "assets/mobileMenu/expand-arrow.png";
 import collapseArrow from "assets/mobileMenu/collapse-arrow.png";
 import { Link } from "react-router-dom";
 import PropTypes from "prop-types";
+import styles from "components/mobileMenu/mobileMenu.module.css";
```

Comment:
to ten sam folder, możesz dać from "./mobileMenu.module.css";

Reply:
w całym projekcie mam absolute imports i nie robiłem "./" jak są w tym samym folderze więc nie wiem czy jest sens zmieniać w jednym miejscu